modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java

```diff
@@ -81,66 +75,15 @@ public DDMStorageAdapterDeleteResponse delete(
 			DDMStorageAdapterDeleteRequest ddmStorageAdapterDeleteRequest)
 		throws StorageException {
 
-		try {
-			long objectEntryId = ddmStorageAdapterDeleteRequest.getPrimaryKey();
-
-			ObjectDefinition objectDefinition = _fetchObjectDefinition(
-				objectEntryId);
-
-			DefaultObjectEntryManager defaultObjectEntryManager =
-				DefaultObjectEntryManagerProvider.provide(
-					_objectEntryManagerRegistry.getObjectEntryManager(
-						objectDefinition.getStorageType()));
-
-			ObjectEntry objectEntry =
-				defaultObjectEntryManager.fetchObjectEntry(
-					_getDTOConverterContext(
-						null, null, LocaleUtil.getSiteDefault()),
-					objectDefinition, objectEntryId);
-
-			if (objectEntry != null) {
-				defaultObjectEntryManager.deleteObjectEntry(
-					objectDefinition, objectEntry.getId());
-			}
-
-			return DDMStorageAdapterDeleteResponse.Builder.newBuilder(
-			).build();
-		}
-		catch (Exception exception) {
-			throw new StorageException(exception);
-		}
+		throw new UnsupportedOperationException();
```

Comment:
Why don't we support delete/get operations anymore?

Reply:
The Report Entries tab is also hidden

Reply:
Thank you for the details!